dataactbroker/handlers/fileHandler.py

```diff
@@ -1343,6 +1344,10 @@ def list_submissions(page, limit, certified, sort='modified', order='desc'):
     """ List submission based on current page and amount to display. If provided, filter based on
     certification status """
     sess = GlobalDB.db().session
+    db = GlobalDB.db()
+    metadata = MetaData(bind=db.engine)
+
+    submission_updated_view = Table('submission_updated_at_view', metadata, autoload=True, info=dict(is_view=True))
```

Comment:
The latter part is what I was thinking as well. Moving this in as a separate file (alongside the models) but a file just for views that we're using. Each view would have it's own class, etc. It'll help with setting up the architecture for future enhancements since it's definitely a useful feature.